### ISSUE TITLE
Local storage support

### ISSUE THREAD

**Issue by MuscleRumble** [cryptomator/cryptomator-ios#21]
Add local storage support. This local storage should also be accessible through iTunes on the desktop computer so that manual synchronization is possible.

**Comment by masvil** [comment]
Don't you think Apple devices native encryption is strong enough?
Maybe we can use any other "Documents Providers" app to store files without the need of one more encryption layer. Just to not bloat the app, I mean.

**Comment by overheadhunter** [comment]
> Just to not bloat the app

I like the way you think 👍

**Comment by MuscleRumble** [comment]
You're right that a local storage defeats the purpose of the app. Our core feature is to secure files in the cloud. So this will probably have a very low priority or maybe even not implemented at all. :wink: I'll keep this open for further discussion, because we might get further insight on use cases we haven't thought of.

Maybe a related feature would be "make files available offline". There is currently no open issue on GitHub for this, even though we have planned this for the future. But when this feature is implemented, it might make this issue obsolete altogether.

**Comment by petermichelson** [comment]
"make files available offline" in combination with "swipe throug files" and "thumbnail preview" would be awesome!

**Comment by ckdanny** [comment]
If support accessing `Files` in the ios would be very useful.

**Comment by cwmke** [comment]
` #You're right that a local storage defeats the purpose of the app.
`

@tobihagemann I sync files between my desktop and phone with syncthing and Mobius Sync. Cryptomater's benefit in this regard is the files are locked except when needed (which is precisely why I use Cryptomator 😉.)
